packages/shared/examples/shaped-scatter-plot/shaped-scatter-plot.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useCallback } from 'react'
+import { Position, Scale, Scatter, colors } from '@unovis/ts'
+import { VisAxis, VisBulletLegend, VisScatter, VisTooltip, VisXYContainer } from '@unovis/react'
+import { data, DataRecord, shapes, categories } from './data'
+
+const shapeScale = Scale.scaleOrdinal(shapes).domain(categories)
+const colorScale = Scale.scaleOrdinal(colors).domain(categories)
+
+export default function ShapedScatterPlot (): JSX.Element {
+  const legendItems = categories.map(v => ({ name: v, shape: shapeScale(v) }))
+  const tooltipTriggers = {
+    [Scatter.selectors.point]: (d: DataRecord) => `
+      <strong>name</strong>: ${d.name} <br>
+      <strong>owner</strong>: ${d.owner} <br>
+      <strong>bn parameters</strong>: ${d.trainedParam}`,
+  }
+
+  function sumCategories (category: string): string {
+    return categories.indexOf(category) === -1 ? 'Other' : category
+  }
+
+  return (
+    <>
+      <h2>The Rise and Rise of A.I. Large Language Models</h2>
+      <VisBulletLegend items={legendItems}/>
+      <VisXYContainer data={data} height={'60vh'} scaleByDomain={true}>
+        <VisScatter
+          x={useCallback((d: DataRecord) => +(new Date(d.date)), [])}
+          y={useCallback((d: DataRecord) => d.trainedParam, [])}
+          shape={useCallback((d: DataRecord) => shapeScale(sumCategories(d.owner)), [])}
+          size={15}
+          label={useCallback((d: DataRecord) => d.name, [])}
+          labelPosition={Position.Bottom}
+          labelHideOverlapping={true}
+          labelColor="grey"
```

Comment:
The `grey` color looks a bit dark in the `dark` mode.

![image](https://raw.githubusercontent.com/f5/unovis/cf972d60437e5a03b1c9ddf0f1ac0f4d9bb75693/packages/shared/examples/_previews/shaped-scatter-plot-dark.png)

Reply:
Good call. Updated to use the default color.